ports/mimxrt10xx/common-hal/microcontroller/Processor.c

```diff
@@ -52,15 +52,13 @@ float common_hal_mcu_processor_get_temperature(void) {
     return temp;
 }
 
-uint32_t common_hal_mcu_processor_set_frequency(mcu_processor_obj_t *self,
+void common_hal_mcu_processor_set_frequency(mcu_processor_obj_t *self,
     uint32_t frequency) {
     uint32_t freq = frequency / 1000000;
     if (freq != 24 && freq != 150 && freq != 396 && freq != 450 && freq != 528 && freq != 600 &&
         freq != 720 && freq != 816 && freq != 912 && freq != 960 && freq != 1008) {
         mp_raise_ValueError(translate("Frequency must be 24, 150, 396, 450, 528, 600, 720, 816, 912, 960 or 1008 Mhz"));
     }
-    SystemCoreClock = setarmclock(frequency);
-    return SystemCoreClock;
```

Comment:
why were these lines removed? doesn't it stop the clock setting from becoming effective?